pycbc/inference/prior.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2016  Christopher M. Biwer
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+"""
+This modules provides classes and functions for evaluating the prior
+for parameter estimation.
+"""
+
+def no_prior(params):
+    """ Function that returns default value if user does not specify a prior.
+    """
+    return 0
+
```

Comment:
Also I don't think we intend for priors to be functions as well. But these generator classes really just return 1 value. If we use ```__call__``` for classes to return the value instead of class functions named ```generate``` or ```prior```, its more general.